apps/aragon-fundraising/app/src/screens/Overview.js

```diff
@@ -29,22 +31,75 @@ export default () => {
+  const [trendBatch, setTrendBatch] = useState(null)
+
+  // *****************************
+  // effects
+  // *****************************
+  // update the base batch to compute trends
+  useEffect(() => {
+    // search the closest batch from (now - 1 month) to create some trends
+    const oneMonthAgo = subMonths(new Date(), 1).getTime()
+    const trendBatch = batches.reduce(
+      (closest, b) => {
+        const currentClosest = Math.abs(closest.timestamp - oneMonthAgo)
+        const current = Math.abs(b.timestamp - oneMonthAgo)
+        return currentClosest < current ? closest : b
+      },
+      { timestamp: new Date() }
+    )
+    setTrendBatch(trendBatch)
+  }, [batches])
+
   // *****************************
   // human readable values
   // *****************************
-  const adjustedTokenSupply = formatBigNumber(realSupply, tokenDecimals)
-  const adjustedReserves = reserveBalance ? formatBigNumber(reserveBalance.minus(toBeClaimed), daiDecimals) : '...'
-  const adjustedMonthlyAllowance = formatBigNumber(toMonthlyAllocation(rate, daiDecimals), daiDecimals)
-  const adjustedMarketCap = price ? formatBigNumber(price.times(realSupply), daiDecimals) : '...'
-  const adjustedPrice = price ? formatBigNumber(price, 0) : '...'
+  //
+  // numbers
+  const adjustedPrice = price ? formatBigNumber(price, 0, { numberPrefix: '$' }) : '...'
+  const marketCap = price ? price.times(realSupply) : null
+  const adjustedMarketCap = price && marketCap ? formatBigNumber(marketCap, daiDecimals, { numberPrefix: '$' }) : '...'
   const tradingVolume = orders
     // only keep DAI orders
     .filter(o => o.collateral === daiAddress)
     // keep values
     .map(o => o.value)
     // sum them and tada, you got the trading volume
     .reduce((acc, current) => acc.plus(current), new BigNumber(0))
-  const adjsutedTradingVolume = formatBigNumber(tradingVolume, daiDecimals)
+  const adjustedTradingVolume = formatBigNumber(tradingVolume, daiDecimals, { numberPrefix: '$' })
+  const adjustedTokenSupply = formatBigNumber(realSupply, tokenDecimals)
+  const realReserve = reserveBalance ? reserveBalance.minus(toBeClaimed) : null
+  const adjustedReserves = realReserve ? formatBigNumber(realReserve, daiDecimals, { numberPrefix: '$' }) : '...'
+  const adjustedMonthlyAllowance = formatBigNumber(toMonthlyAllocation(rate, daiDecimals), daiDecimals, { numberPrefix: '$' })
+  const adjustedYearlyAllowance = formatBigNumber(toMonthlyAllocation(rate, daiDecimals).times(12), daiDecimals, { numberPrefix: '$' })
+  //
+  // trends
+  const adjustedPriceTrend =
+    price && trendBatch?.startPrice ? formatBigNumber(price.minus(trendBatch.startPrice), 0, { keepSign: true, numberPrefix: '$' }) : null
+  // if startPrice is here, realSupply too, since NewMetaBatch event occurs before NewBatch one
+  const marketCapDiff = marketCap && trendBatch?.startPrice ? marketCap.minus(trendBatch.startPrice.times(trendBatch.realSupply)) : null
```

Comment:
Well, yes, but after some investigation, there is a small problem.
Since the formula to get the percentage variation is the following: 
`((end - start) / start) * 100`

As soon as you try to calculate a variation with a starting value where `start = 0` you get `Infinity` everywhere, and that's pretty boring :(

Reply:
So I added the pct variation only for the price since it's starting value cannot be 0.